src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -0,0 +1,92 @@
+
+Read this page regarding [setting up structured logging](monitor-logging.html#Guice_products_and_logging).
+
+We recommend closely monitoring **ERROR** and **WARNING** logs. Those logs should be considered not normal.
+
+If you encounter some suspicious logs:
+ - If you have any doubt about the log being caused by a bug in James source code, please reach us via 
+ [the bug tracker](https://issues.apache.org/jira/browse/JAMES) or [the user mailing list](/mail.html)
+ - They can be due to insufficient performance from tier applications (eg Cassandra timeouts). In such case we advise
+ you a close review of performances at the database level.
+
+Leveraging filters in Kibana discover view can help filtering out "already known" frequently occurring logs.
+
+When reporting logs error or warning, consider adding the full logs, and related data (eg EML triggering an issue) to
+the bug report in order to ease resolution.
+
+### Metrics graphs into Grafana
+
+James keeps tracks of various metrics and allow to easily visualize them.
+
+Read this page for [explanations on metrics](metrics.html).
+
+Here is a list of [available metric boards](https://github.com/apache/james-project/tree/master/grafana-reporting)
+
+Configuration of [ElasticSearch metric exporting](config-elasticsearch.html) allows a direct display within 
+[Grafana](https://grafana.com/)
+
+Monitoring these graphs on a regular basis allow diagnosing early some performance issues. 
+
+If some metrics seems abnormally slow despite in depth database performance tuning, feedback is appreciated as well on 
+[the bug tracker](https://issues.apache.org/jira/browse/JAMES) or [the user mailing list](/mail.html). Any additional 
+details categorizing the slowness is appreciated as well (details of the slow requests for instance).
+
+### WebAdmin HealthChecks
+
+James webadmin API allow to run healthChecks for a quick health overview.
+
+Here is related [webadmin documentation](manage-webadmin.html#HealthCheck)
+
+Here are the available checks alongs side the insight they offer:
+
+ - **Cassandra backend**: Cassandra storage. Ensure queries can be executed on the connection James uses.
+ - **ElasticSearch Backend**: ElasticSearch storage. Triggers an ElasticSearch health request on indices James uses.
+ - **RabbitMQ backend**: RabbitMQ messaging. Verifies an open connection and an open channel are well available.
+ - **Guice application lifecycle**: Ensures James Guice successfully started, and is up. Logs should contain explanations if
+ James did not start well.
+ - **MessageFastViewProjection**: Follows MessageFastViewProjection cache miss rates and warns if it is below 10%. If this 
+ projection is missing, this results in performance issues for JMAP GetMessages list requests. WebAdmin offers a
+ [global](manage-webadmin.html#recomputing-global-jmap-fast-message-view-projection) and 
+ [per user](manage-webadmin.html#recomputing-user-jmap-fast-message-view-projection) projection re-computation.
```

Comment:
explanation why cache could be below 10% for example after an imap sync, or in some non jmap cases is missing

Reply:
I don't understand this comment.

I don't want to over-charge this section.

If it requires more attention, we can have a `managing JMAP` section?

Reply:
my point is that you can very easily have a warning, even with a perfectly working server in a perfectly valid scenario. You just need to explain that this can happens, and give an example of why.